File: PrimeNumbers.java
import java.util.Scanner;

public class PrimeNumbers {
        static boolean isPrime(int num) {
        if (num < 2) {
            return false;
        }
        for (int i = 2; i * i <= num; i++) {
            if (num % i == 0) {
                return false;
            }
        }
        return true;
    }

    static void printPrimesInRange(int start, int end) {
        System.out.println("Prime Numbers:");
        for (int i = start; i <= end; i++) {
            if (isPrime(i)) {
                System.out.println(i);
            }
        }
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the first range: ");
        int start = scanner.nextInt();

        System.out.print("Enter the last range: ");
        int end = scanner.nextInt();

        System.out.println();
        printPrimesInRange(start, end);

        scanner.close();
    }
}

File: PrimeNumbers.py
def is_prime(num):
    if num < 2:
        return False
    for i in range(2, int(num ** 0.5) + 1):
        if num % i == 0:
            return False
    return True

def print_primes_in_range(start, end):
    print("Prime Numbers:")
    for num in range(start, end + 1):
        if is_prime(num):
            print(num)

start = int(input("Enter the first range: "))
end = int(input("Enter the last range: "))

print()
print_primes_in_range(start, end)
